compat-chests.lua

```diff
@@ -152,17 +161,21 @@ elseif minetest.get_modpath("hades_chests") then
 		-- get the fields from the chest formspec, we can do this bc. newest functions are called first
 		-- https://github.com/minetest/minetest/blob/d4b10db998ebeb689b3d27368e30952a42169d03/doc/lua_api.md?plain=1#L5840
 		minetest.register_on_player_receive_fields(function(player, formname, fields)
-			if formname == "hades_chests:chest_locked" then
-				local pn = player:get_player_name()
-				local pos = open_chests[pn]
-				if not fields.quit and pos and pipeworks.may_configure(pos, player) then
-					-- Pipeworks Switch
-					fs_helpers.on_receive_fields(pos, fields)
-					minetest.show_formspec(pn, "hades_chests:chest_locked", get_locked_chest_formspec(pos))
-				end
-				-- Do NOT return true here, the callback from hades still needs to run (if they add one)
-				return false
+			if formname ~= "hades_chests:chest_locked" then
+				return
 			end
+			local pn = player:get_player_name()
+			if not pn then
+				return
+			end
```

Comment:
This is almost sure unnecessary and could be removed.

Reply:
is removed in https://github.com/mt-mods/pipeworks/pull/125/commits/7f63b3f6bf384a325c07ce309f760f06502e212a